### DISCUSSION TITLE
Run failed. Kfp v2

### DISCUSSION THREAD

**Discussion by oscherepanov** [kubeflow/pipelines#10772] [General]
I created a simple pipeline:

![Снимок экрана_2024-05-02_16-45-37](https://github.com/kubeflow/pipelines/assets/6450571/b2e11956-3618-42e9-831b-90240655d5cc)

Pod logs:
I0502 11:26:30.659633      20 driver.go:153] PipelineRoot="minio://mlpipeline/v2/artifacts" from default config
F0502 11:26:30.713634      20 main.go:79] KFP driver: driver.RootDAG(pipelineName=test, runID=faa351ec-2328-4530-910e-c71ee87c541b, runtimeConfig, componentSpec) failed: Failed PutContext(name="test", type="system.Pipeline", typeid=10): rpc error: code = NotFound desc = Cannot find type for type_id: 10 name: "test"No type found for query, type_id: 10
time="2024-05-02T11:26:31.220Z" level=info msg="sub-process exited" argo=true error="<nil>"
time="2024-05-02T11:26:31.233Z" level=error msg="cannot save parameter /tmp/outputs/execution-id" argo=true error="open /tmp/outputs/execution-id: no such file or directory"
time="2024-05-02T11:26:31.233Z" level=error msg="cannot save parameter /tmp/outputs/iteration-count" argo=true error="open /tmp/outputs/iteration-count: no such file or directory"
time="2024-05-02T11:26:31.233Z" level=error msg="cannot save parameter /tmp/outputs/condition" argo=true error="open /tmp/outputs/condition: no such file or directory"

**Comment by rimolive** [comment]
For now ignore these errors:
```
time="2024-05-02T11:26:31.233Z" level=error msg="cannot save parameter /tmp/outputs/execution-id" argo=true error="open /tmp/outputs/execution-id: no such file or directory"
time="2024-05-02T11:26:31.233Z" level=error msg="cannot save parameter /tmp/outputs/iteration-count" argo=true error="open /tmp/outputs/iteration-count: no such file or directory"
time="2024-05-02T11:26:31.233Z" level=error msg="cannot save parameter /tmp/outputs/condition" argo=true error="open /tmp/outputs/condition: no such file or directory"
```
These errors are covered by #9678 and #10729.

As for this error:
```
F0502 11:26:30.713634 20 main.go:79] KFP driver: driver.RootDAG(pipelineName=test, runID=faa351ec-2328-4530-910e-c71ee87c541b, runtimeConfig, componentSpec) failed: Failed PutContext(name="test", type="system.Pipeline", typeid=10): rpc error: code = NotFound desc = Cannot find type for type_id: 10 name: "test"No type found for query, type_id: 10
```
I used the same code as your, compiled using KFP SDK 2.7.0 and I could run in my backed running KFP 2.0.5.

Could you tell me more about your installation? What is the backend KFP and KFP SDK versions?
